hack/jenkins/common.sh

```diff
@@ -91,7 +91,7 @@ else
 fi
 
 # installing golang so we could do go get for gopogh
-./installers/check_install_golang.sh "1.17" "/usr/local" || true
```

Comment:
this will need to be updated in our Update Golang automation too

Reply:
done, removed from update_golang_version.go